modules/ravestate/testfixtures.py

```diff
@@ -23,7 +23,7 @@
 def state_fixture(mocker):
     @state(write=(DEFAULT_PROPERTY_ID,), read=(DEFAULT_PROPERTY_ID,))
     def state_mock_fn(ctx):
-        pass
+        ctx[DEFAULT_PROPERTY_ID] = "test"
```

Comment:
Is this used anywhere yet?
Maybe we can replace the hardcoded value with a variable

Reply:
Not sure about a lot of things in testfixtures.py, let's keep it on our radar.